cmd/pitr/recoverer/recoverer.go

```diff
@@ -136,16 +137,34 @@ func getStartGTIDSet(c BackupS3) (string, error) {
 	if err != nil {
 		return "", errors.Wrap(err, "new storage manager")
 	}
-
-	infoObj, err := s3.GetObject("xtrabackup_info.00000000000000000000") //TODO: work with compressed file
+	listInfo, err := s3.ListObjects("xtrabackup_info")
 	if err != nil {
-		return "", errors.Wrapf(err, "get %s info", prefix)
+		return "", errors.Wrapf(err, "list %s info fies", prefix)
 	}
+	sort.Strings(listInfo)
 
-	lastGTID, err := getLastBackupGTID(infoObj)
-	if err != nil {
-		return "", errors.Wrap(err, "get last backup gtid")
+	var lastGTID string
+	for _, fileName := range listInfo {
+		format := "txt"
+		infoObj, err := s3.GetObject(fileName)
+		if err != nil {
+			return "", errors.Wrapf(err, "get %s info", prefix)
+		}
+
+		if strings.Contains(fileName, "lz4") {
```

Comment:
if it's a format name, i think it's worth to check with `strings.HasSuffix`

Reply:
this files names have format like `backup_info.lz4.0000000000001`

Reply:
and we don't need format anymore https://github.com/percona/percona-xtradb-cluster-operator/pull/743/commits/acb9a424f577f3a06fdf388b2fdce0775c2157a9